src/ExpressionEvaluator/Core/Source/ResultProvider/NetFX20/Helpers/Placeholders.cs

```diff
@@ -103,7 +103,8 @@ public TargetFrameworkAttribute(string frameworkName)
 
 namespace System.Threading
 {
-    public struct CancellationToken
+    public readonly struct CancellationToken
     {
+        public bool IsCancellationRequested => false;
```

Comment:
This changes the semantics of the code. Previously, on NET20, it was `exception is OperationCanceledException`.

Now, it's:

```csharp
exception is OperationCanceledException && cancellationToken.IsCancellationRequested;
```

which is false.

Reply:
@sharwell is it possible to test this to make sure that this would fail a unit test?

Reply:
> This changes the semantics of the code

This is fine. The new behavior is correct for the semantics of the method and also isn't used because all the callers are newer TFM.

>  is it possible to test this to make sure that this would fail a unit test?

I'm not sure what this means.

Reply:
> This is fine. The new behavior is correct for the semantics of the method

Is this true? It certainly doesn't look like it. It looks like it will always be false, no matter what. If that's fine, please include a comment as to why that's fine.

> and also isn't used because all the callers are newer TFM.

Then should we just condition the method on that newer TFM?

> > is it possible to test this to make sure that this would fail a unit test?
>
> I'm not sure what this means.

Naively, it looks like you've introduced a bug. If that's true, and it wasn't going to fail a unit test, then I'd like to to be unit tested :).

Reply:
I updated to add a comment. A test was not added because the implementation is trivially correct: .NET Framework 2.0 does not support `CancellationToken` and does not support cancellation _via_ a `CancellationToken`. Therefore, when one asks if cancellation has been requested via a `CancellationToken`, the answer is trivially no.